src/app/component/user/auth/sign-up/sign-up.component.ts

```diff
@@ -41,10 +59,14 @@ export class SignUpComponent implements OnInit {
             this.firstNameErrorMessageBackEnd = error.message;
           } else if (error.name === 'lastName') {
             this.lastNameErrorMessageBackEnd = error.message;
+          } else if (error.name === 'userName') {
```

Comment:
maybe to use  switch case

Reply:
was changed      https://github.com/ita-social-projects/GreenCityClient/pull/159/commits/020f38d04c6332d80771ab089fc3e792f6125ed7#diff-52b1610693fc50f73ef687a5698e20ba